thrift/src/test/java/com/linecorp/armeria/it/metric/PrometheusMetricsIntegrationTest.java

```diff
@@ -140,6 +143,7 @@ private static void hello_first_endpoint() throws Exception {
                           "client_total_duration_seconds_sum{handler=\"Foo\","));
 
         final String content = makeMetricsRequest().content().toStringUtf8();
+        logger.debug("Metrics reported by the exposition service:\n{}", content);
```

Comment:
Not sure but can we remove this debug print?

Reply:
We can, but it's useful for debugging. Gradle will not print it as long as the test passes.